docs/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "docs"
+version = "0.1.0"
+description = ""
+authors = ["Lune <eng@lune.co>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+sphinx = "^7.2.6"
+sphinx-rtd-theme = "^1.2.2"
+
+
```

Comment:
nitpick: there's two empty lines here instead of one 🙈

Reply:
Fixing, good spot